googletest/include/gtest/internal/gtest-internal.h

```diff
@@ -1338,17 +1353,21 @@ class FlatTuple
 
 #define GTEST_TEST_NO_THROW_(statement, fail) \
   GTEST_AMBIGUOUS_ELSE_BLOCKER_ \
-  if (::testing::internal::AlwaysTrue()) { \
+  if (::testing::internal::AdditionalMessage message = ".") { \
```

Comment:
Lets add a comment to describe why this is here"
We need declare message in the scope of the if. It can't go into the body of the if because then we cannot access it in the else. It also can't go before the if block because than we have to make up some very unique name (line number is not enough)."